### PR TITLE
M4bulmagd remove hls

### PR DESCRIPTION
**Pull request by m4bulmagd**

### CODE REVIEW

**Comment by m4bulmagd** [comment]
this test passed locally but failed here because it checked for files in a storage bucket. any idea how it work here ?

File: backend/src/labelit/utils/audio_utils.py [hunk 1]
```diff
@@ -148,215 +82,50 @@ def generate_dynamic_waveform_from_audio(input_file, output_file, create_folder=
...
+            head_obj = None
 
-        convert_and_upload_single_file(
-            audio_file_name=audio_file_name,
-            hls_file_name=hls_file_name,
-            hls_file_key=hls_file_key,
-            audio_file_key=audio_file_key,
-            generate_waveform=True,
+        MIN_AUDIO_SIZE_TO_GENERATE_PEAKS = int(
+            os.getenv("MIN_AUDIO_SIZE_TO_GENERATE_PEAKS", 4)
```
Comment (corentingiraud) [inline backend/src/labelit/utils/audio_utils.py:110]: settings.py might be a better place for this ?

File: docker-compose.yaml [hunk 2]
```diff
@@ -26,7 +26,8 @@ services:
       - postgres
 
   minio:
-    image: minio/minio:RELEASE.2023-01-02T09-40-09Z
+    image: minio/minio:RELEASE.2022-10-24T18-35-07Z.hotfix.ce525fdaf
+    #image: minio/minio:RELEASE.2023-01-02T09-40-09Z #had some issue with this ver
```
Comment (corentingiraud) [inline docker-compose.yaml:30]: Keep the newer version in the repository, even if in your local system you use a different one.

**Review by corentingiraud — CHANGES_REQUESTED** [review]
Very good work ! As a lot of this has been removed, we might deploy in staging first to check if the project work as expected :)

File: docker-compose.yaml (hunk 2 above)
Comment (corentingiraud) [reply docker-compose.yaml:30]: By the way, what problem do you have with the new version ?
You might need to delete the docker volume.

**Review by corentingiraud — APPROVED** [review]
LGTM 👍

**Pull request merged**